Makefile

```diff
@@ -0,0 +1,7 @@
+gen: swagger.json
+	@ kuberl_gen
```

Comment:
Did you forget to add a file to the commit? I don't see a kuberl_gen anywhere.

Reply:
`kuberl_gen` comes from here: https://github.com/tsloughter/kuberl/blob/1d4f48b0f85c2938f78542f36ee0b1b18889cee3/shell.nix#L4-L17